proto/buf/alpha/registry/v1alpha1/reference.proto

```diff
@@ -63,7 +63,7 @@ message RepositoryVCSCommit {
   // immutable
   google.protobuf.Timestamp create_time = 2;
   // The name of the VCS commit, e.g. for Git, it would be the Git hash.
-  string hash = 3;
+  string name = 3;
```

Comment:
This was initially name, we changed it to hash because that better represented what it was, but I changed it back to `name` because upon integration in `core` I found out that the frontend expects all references to have a `name`. I'd rather we go through and refactor that as part of the v1beta1 effort, if we need to, so changing it back for now.